Grant project viewer to staging project writers & fake-prod writers

thockin opened the pull request:
This allows the UI to work.

@justaugustus

justaugustus commented:
/lgtm
/approve

dims commented:
/lgtm

@thockin you are going to run it? or do you want me to?

thanks,
Dims

dims commented:
/approve
/hold

hold to actually run the script

thockin commented:
I will run it

thockin commented:
staging:

```
thockin@freakshow2 k8s.io master /infra/gcp$ ./ensure-staging-storage.sh 
build-image
Ensuring project exists: k8s-staging-build-image
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-staging-build-image/billingInfo
projectId: k8s-staging-build-image
Empowering k8s-infra-staging-build-image@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-build-image].
bindings:
- members:
  - serviceAccount:service-960211007710@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-build-image@kubernetes.io
  role: roles/viewer
etag: BwWNWqM2vOw=
version: 1
Enabling the container registry API
Ensuring the registry exists and is readable
No changes made to gs://artifacts.k8s-staging-build-image.appspot.com/
Enabling Bucket Policy Only for gs://artifacts.k8s-staging-build-image.appspot.com...
Empowering GCR admins
Updated IAM policy for project [k8s-staging-build-image].
bindings:
- members:
  - serviceAccount:service-960211007710@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-build-image@kubernetes.io
  role: roles/viewer
etag: BwWNWqOVqsg=
version: 1
No changes made to gs://artifacts.k8s-staging-build-image.appspot.com/
No changes made to gs://artifacts.k8s-staging-build-image.appspot.com/
Empowering k8s-infra-staging-build-image@kubernetes.io to GCR
No changes made to gs://artifacts.k8s-staging-build-image.appspot.com/
No changes made to gs://artifacts.k8s-staging-build-image.appspot.com/
Enabling the GCS API
Ensuring the bucket exists and is world readable
No changes made to gs://k8s-staging-build-image/
Enabling Bucket Policy Only for gs://k8s-staging-build-image...
Ensuring the bucket has retention of 30d
Setting Retention Policy on gs://k8s-staging-build-image/...
Ensuring the bucket has auto-deletion of 60 days
Setting lifecycle configuration on gs://k8s-staging-build-image/...
Empowering GCS admins
Updated IAM policy for project [k8s-staging-build-image].
bindings:
- members:
  - serviceAccount:service-960211007710@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-build-image@kubernetes.io
  role: roles/viewer
etag: BwWNWqRemRU=
version: 1
No changes made to gs://k8s-staging-build-image/
No changes made to gs://k8s-staging-build-image/
Empowering k8s-infra-staging-build-image@kubernetes.io to GCS
No changes made to gs://k8s-staging-build-image/
No changes made to gs://k8s-staging-build-image/
Done
cip-test
Ensuring project exists: k8s-staging-cip-test
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-staging-cip-test/billingInfo
projectId: k8s-staging-cip-test
Empowering k8s-infra-staging-cip-test@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-cip-test].
bindings:
- members:
  - serviceAccount:service-324460563566@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:324460563566-compute@developer.gserviceaccount.com
  - serviceAccount:324460563566@cloudservices.gserviceaccount.com
  - serviceAccount:service-324460563566@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-cip-test@kubernetes.io
  role: roles/viewer
etag: BwWNWqTqCfw=
version: 1
Enabling the container registry API
Ensuring the registry exists and is readable
No changes made to gs://artifacts.k8s-staging-cip-test.appspot.com/
Enabling Bucket Policy Only for gs://artifacts.k8s-staging-cip-test.appspot.com...
Empowering GCR admins
Updated IAM policy for project [k8s-staging-cip-test].
bindings:
- members:
  - serviceAccount:service-324460563566@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:324460563566-compute@developer.gserviceaccount.com
  - serviceAccount:324460563566@cloudservices.gserviceaccount.com
  - serviceAccount:service-324460563566@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-cip-test@kubernetes.io
  role: roles/viewer
etag: BwWNWqVBPIY=
version: 1
No changes made to gs://artifacts.k8s-staging-cip-test.appspot.com/
No changes made to gs://artifacts.k8s-staging-cip-test.appspot.com/
Empowering k8s-infra-staging-cip-test@kubernetes.io to GCR
No changes made to gs://artifacts.k8s-staging-cip-test.appspot.com/
No changes made to gs://artifacts.k8s-staging-cip-test.appspot.com/
Enabling the GCS API
Ensuring the bucket exists and is world readable
No changes made to gs://k8s-staging-cip-test/
Enabling Bucket Policy Only for gs://k8s-staging-cip-test...
Ensuring the bucket has retention of 30d
Setting Retention Policy on gs://k8s-staging-cip-test/...
Ensuring the bucket has auto-deletion of 60 days
Setting lifecycle configuration on gs://k8s-staging-cip-test/...
Empowering GCS admins
Updated IAM policy for project [k8s-staging-cip-test].
bindings:
- members:
  - serviceAccount:service-324460563566@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:324460563566-compute@developer.gserviceaccount.com
  - serviceAccount:324460563566@cloudservices.gserviceaccount.com
  - serviceAccount:service-324460563566@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-cip-test@kubernetes.io
  role: roles/viewer
etag: BwWNWqYNjnk=
version: 1
No changes made to gs://k8s-staging-cip-test/
No changes made to gs://k8s-staging-cip-test/
Empowering k8s-infra-staging-cip-test@kubernetes.io to GCS
No changes made to gs://k8s-staging-cip-test/
No changes made to gs://k8s-staging-cip-test/
Done
cluster-api
Ensuring project exists: k8s-staging-cluster-api
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-staging-cluster-api/billingInfo
projectId: k8s-staging-cluster-api
Empowering k8s-infra-staging-cluster-api@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-cluster-api].
bindings:
- members:
  - serviceAccount:service-190130481896@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:190130481896-compute@developer.gserviceaccount.com
  - serviceAccount:190130481896@cloudservices.gserviceaccount.com
  - serviceAccount:service-190130481896@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-cluster-api@kubernetes.io
  role: roles/viewer
etag: BwWNWqaWWDI=
version: 1
Enabling the container registry API
Ensuring the registry exists and is readable
No changes made to gs://artifacts.k8s-staging-cluster-api.appspot.com/
Enabling Bucket Policy Only for gs://artifacts.k8s-staging-cluster-api.appspot.com...
Empowering GCR admins
Updated IAM policy for project [k8s-staging-cluster-api].
bindings:
- members:
  - serviceAccount:service-190130481896@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:190130481896-compute@developer.gserviceaccount.com
  - serviceAccount:190130481896@cloudservices.gserviceaccount.com
  - serviceAccount:service-190130481896@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-cluster-api@kubernetes.io
  role: roles/viewer
etag: BwWNWqbxTAo=
version: 1
No changes made to gs://artifacts.k8s-staging-cluster-api.appspot.com/
No changes made to gs://artifacts.k8s-staging-cluster-api.appspot.com/
Empowering k8s-infra-staging-cluster-api@kubernetes.io to GCR
No changes made to gs://artifacts.k8s-staging-cluster-api.appspot.com/
No changes made to gs://artifacts.k8s-staging-cluster-api.appspot.com/
Enabling the GCS API
Ensuring the bucket exists and is world readable
No changes made to gs://k8s-staging-cluster-api/
Enabling Bucket Policy Only for gs://k8s-staging-cluster-api...
Ensuring the bucket has retention of 30d
Setting Retention Policy on gs://k8s-staging-cluster-api/...
Ensuring the bucket has auto-deletion of 60 days
Setting lifecycle configuration on gs://k8s-staging-cluster-api/...
Empowering GCS admins
Updated IAM policy for project [k8s-staging-cluster-api].
bindings:
- members:
  - serviceAccount:service-190130481896@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:190130481896-compute@developer.gserviceaccount.com
  - serviceAccount:190130481896@cloudservices.gserviceaccount.com
  - serviceAccount:service-190130481896@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-cluster-api@kubernetes.io
  role: roles/viewer
etag: BwWNWqe43Wo=
version: 1
No changes made to gs://k8s-staging-cluster-api/
No changes made to gs://k8s-staging-cluster-api/
Empowering k8s-infra-staging-cluster-api@kubernetes.io to GCS
No changes made to gs://k8s-staging-cluster-api/
No changes made to gs://k8s-staging-cluster-api/
Done
cluster-api-aws
Ensuring project exists: k8s-staging-cluster-api-aws
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-staging-cluster-api-aws/billingInfo
projectId: k8s-staging-cluster-api-aws
Empowering k8s-infra-staging-cluster-api-aws@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-cluster-api-aws].
bindings:
- members:
  - serviceAccount:service-433651898792@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-cluster-api-aws@kubernetes.io
  role: roles/viewer
etag: BwWNWqg7gkA=
version: 1
Enabling the container registry API
Ensuring the registry exists and is readable
No changes made to gs://artifacts.k8s-staging-cluster-api-aws.appspot.com/
Enabling Bucket Policy Only for gs://artifacts.k8s-staging-cluster-api-aws.appspot.com...
Empowering GCR admins
Updated IAM policy for project [k8s-staging-cluster-api-aws].
bindings:
- members:
  - serviceAccount:service-433651898792@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-cluster-api-aws@kubernetes.io
  role: roles/viewer
etag: BwWNWqiXDJ4=
version: 1
No changes made to gs://artifacts.k8s-staging-cluster-api-aws.appspot.com/
No changes made to gs://artifacts.k8s-staging-cluster-api-aws.appspot.com/
Empowering k8s-infra-staging-cluster-api-aws@kubernetes.io to GCR
No changes made to gs://artifacts.k8s-staging-cluster-api-aws.appspot.com/
No changes made to gs://artifacts.k8s-staging-cluster-api-aws.appspot.com/
Enabling the GCS API
Ensuring the bucket exists and is world readable
No changes made to gs://k8s-staging-cluster-api-aws/
Enabling Bucket Policy Only for gs://k8s-staging-cluster-api-aws...
Ensuring the bucket has retention of 30d
Setting Retention Policy on gs://k8s-staging-cluster-api-aws/...
Ensuring the bucket has auto-deletion of 60 days
Setting lifecycle configuration on gs://k8s-staging-cluster-api-aws/...
Empowering GCS admins
Updated IAM policy for project [k8s-staging-cluster-api-aws].
bindings:
- members:
  - serviceAccount:service-433651898792@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-cluster-api-aws@kubernetes.io
  role: roles/viewer
etag: BwWNWqlkREA=
version: 1
No changes made to gs://k8s-staging-cluster-api-aws/
No changes made to gs://k8s-staging-cluster-api-aws/
Empowering k8s-infra-staging-cluster-api-aws@kubernetes.io to GCS
No changes made to gs://k8s-staging-cluster-api-aws/
No changes made to gs://k8s-staging-cluster-api-aws/
Done
coredns
Ensuring project exists: k8s-staging-coredns
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-staging-coredns/billingInfo
projectId: k8s-staging-coredns
Empowering k8s-infra-staging-coredns@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-coredns].
bindings:
- members:
  - serviceAccount:service-848617618266@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-848617618266@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-848617618266@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:848617618266-compute@developer.gserviceaccount.com
  - serviceAccount:848617618266@cloudservices.gserviceaccount.com
  - serviceAccount:service-848617618266@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-coredns@kubernetes.io
  role: roles/viewer
etag: BwWNWqnm52c=
version: 1
Enabling the container registry API
Ensuring the registry exists and is readable
No changes made to gs://artifacts.k8s-staging-coredns.appspot.com/
Enabling Bucket Policy Only for gs://artifacts.k8s-staging-coredns.appspot.com...
Empowering GCR admins
Updated IAM policy for project [k8s-staging-coredns].
bindings:
- members:
  - serviceAccount:service-848617618266@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-848617618266@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-848617618266@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:848617618266-compute@developer.gserviceaccount.com
  - serviceAccount:848617618266@cloudservices.gserviceaccount.com
  - serviceAccount:service-848617618266@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-coredns@kubernetes.io
  role: roles/viewer
etag: BwWNWqo_l9Q=
version: 1
No changes made to gs://artifacts.k8s-staging-coredns.appspot.com/
No changes made to gs://artifacts.k8s-staging-coredns.appspot.com/
Empowering k8s-infra-staging-coredns@kubernetes.io to GCR
No changes made to gs://artifacts.k8s-staging-coredns.appspot.com/
No changes made to gs://artifacts.k8s-staging-coredns.appspot.com/
Enabling the GCS API
Ensuring the bucket exists and is world readable
No changes made to gs://k8s-staging-coredns/
Enabling Bucket Policy Only for gs://k8s-staging-coredns...
Ensuring the bucket has retention of 30d
Setting Retention Policy on gs://k8s-staging-coredns/...
Ensuring the bucket has auto-deletion of 60 days
Setting lifecycle configuration on gs://k8s-staging-coredns/...
Empowering GCS admins
Updated IAM policy for project [k8s-staging-coredns].
bindings:
- members:
  - serviceAccount:service-848617618266@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-848617618266@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-848617618266@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:848617618266-compute@developer.gserviceaccount.com
  - serviceAccount:848617618266@cloudservices.gserviceaccount.com
  - serviceAccount:service-848617618266@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-coredns@kubernetes.io
  role: roles/viewer
etag: BwWNWqsHfGw=
version: 1
No changes made to gs://k8s-staging-coredns/
No changes made to gs://k8s-staging-coredns/
Empowering k8s-infra-staging-coredns@kubernetes.io to GCS
No changes made to gs://k8s-staging-coredns/
No changes made to gs://k8s-staging-coredns/
Done
csi
Ensuring project exists: k8s-staging-csi
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-staging-csi/billingInfo
projectId: k8s-staging-csi
Empowering k8s-infra-staging-csi@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-csi].
bindings:
- members:
  - serviceAccount:service-874328413592@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:874328413592-compute@developer.gserviceaccount.com
  - serviceAccount:874328413592@cloudservices.gserviceaccount.com
  - serviceAccount:service-874328413592@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-csi@kubernetes.io
  role: roles/viewer
etag: BwWNWquPKs4=
version: 1
Enabling the container registry API
Ensuring the registry exists and is readable
No changes made to gs://artifacts.k8s-staging-csi.appspot.com/
Enabling Bucket Policy Only for gs://artifacts.k8s-staging-csi.appspot.com...
Empowering GCR admins
Updated IAM policy for project [k8s-staging-csi].
bindings:
- members:
  - serviceAccount:service-874328413592@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:874328413592-compute@developer.gserviceaccount.com
  - serviceAccount:874328413592@cloudservices.gserviceaccount.com
  - serviceAccount:service-874328413592@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-csi@kubernetes.io
  role: roles/viewer
etag: BwWNWqvo0AI=
version: 1
No changes made to gs://artifacts.k8s-staging-csi.appspot.com/
No changes made to gs://artifacts.k8s-staging-csi.appspot.com/
Empowering k8s-infra-staging-csi@kubernetes.io to GCR
No changes made to gs://artifacts.k8s-staging-csi.appspot.com/
No changes made to gs://artifacts.k8s-staging-csi.appspot.com/
Enabling the GCS API
Ensuring the bucket exists and is world readable
No changes made to gs://k8s-staging-csi/
Enabling Bucket Policy Only for gs://k8s-staging-csi...
Ensuring the bucket has retention of 30d
Setting Retention Policy on gs://k8s-staging-csi/...
Ensuring the bucket has auto-deletion of 60 days
Setting lifecycle configuration on gs://k8s-staging-csi/...
Empowering GCS admins
Updated IAM policy for project [k8s-staging-csi].
bindings:
- members:
  - serviceAccount:service-874328413592@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:874328413592-compute@developer.gserviceaccount.com
  - serviceAccount:874328413592@cloudservices.gserviceaccount.com
  - serviceAccount:service-874328413592@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-csi@kubernetes.io
  role: roles/viewer
etag: BwWNWqyw4fw=
version: 1
No changes made to gs://k8s-staging-csi/
No changes made to gs://k8s-staging-csi/
Empowering k8s-infra-staging-csi@kubernetes.io to GCS
No changes made to gs://k8s-staging-csi/
No changes made to gs://k8s-staging-csi/
Done
kops
Ensuring project exists: k8s-staging-kops
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-staging-kops/billingInfo
projectId: k8s-staging-kops
Empowering k8s-infra-staging-kops@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-kops].
bindings:
- members:
  - serviceAccount:service-889470918518@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-kops@kubernetes.io
  role: roles/viewer
etag: BwWNWq1Bso4=
version: 1
Enabling the container registry API
Ensuring the registry exists and is readable
No changes made to gs://artifacts.k8s-staging-kops.appspot.com/
Enabling Bucket Policy Only for gs://artifacts.k8s-staging-kops.appspot.com...
Empowering GCR admins
Updated IAM policy for project [k8s-staging-kops].
bindings:
- members:
  - serviceAccount:service-889470918518@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-kops@kubernetes.io
  role: roles/viewer
etag: BwWNWq2YumQ=
version: 1
No changes made to gs://artifacts.k8s-staging-kops.appspot.com/
No changes made to gs://artifacts.k8s-staging-kops.appspot.com/
Empowering k8s-infra-staging-kops@kubernetes.io to GCR
No changes made to gs://artifacts.k8s-staging-kops.appspot.com/
No changes made to gs://artifacts.k8s-staging-kops.appspot.com/
Enabling the GCS API
Ensuring the bucket exists and is world readable
No changes made to gs://k8s-staging-kops/
Enabling Bucket Policy Only for gs://k8s-staging-kops...
Ensuring the bucket has retention of 30d
Setting Retention Policy on gs://k8s-staging-kops/...
Ensuring the bucket has auto-deletion of 60 days
Setting lifecycle configuration on gs://k8s-staging-kops/...
Empowering GCS admins
Updated IAM policy for project [k8s-staging-kops].
bindings:
- members:
  - serviceAccount:service-889470918518@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-kops@kubernetes.io
  role: roles/viewer
etag: BwWNWq5fCSs=
version: 1
No changes made to gs://k8s-staging-kops/
No changes made to gs://k8s-staging-kops/
Empowering k8s-infra-staging-kops@kubernetes.io to GCS
No changes made to gs://k8s-staging-kops/
No changes made to gs://k8s-staging-kops/
Done
publishing-bot
Ensuring project exists: k8s-staging-publishing-bot
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-staging-publishing-bot/billingInfo
projectId: k8s-staging-publishing-bot
Empowering k8s-infra-staging-publishing-bot@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-publishing-bot].
bindings:
- members:
  - serviceAccount:service-438481731081@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-publishing-bot@kubernetes.io
  role: roles/viewer
etag: BwWNWq7cCGY=
version: 1
Enabling the container registry API
Ensuring the registry exists and is readable
No changes made to gs://artifacts.k8s-staging-publishing-bot.appspot.com/
Enabling Bucket Policy Only for gs://artifacts.k8s-staging-publishing-bot.appspot.com...
Empowering GCR admins
Updated IAM policy for project [k8s-staging-publishing-bot].
bindings:
- members:
  - serviceAccount:service-438481731081@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-publishing-bot@kubernetes.io
  role: roles/viewer
etag: BwWNWq80f0M=
version: 1
No changes made to gs://artifacts.k8s-staging-publishing-bot.appspot.com/
No changes made to gs://artifacts.k8s-staging-publishing-bot.appspot.com/
Empowering k8s-infra-staging-publishing-bot@kubernetes.io to GCR
No changes made to gs://artifacts.k8s-staging-publishing-bot.appspot.com/
No changes made to gs://artifacts.k8s-staging-publishing-bot.appspot.com/
Enabling the GCS API
Ensuring the bucket exists and is world readable
No changes made to gs://k8s-staging-publishing-bot/
Enabling Bucket Policy Only for gs://k8s-staging-publishing-bot...
Ensuring the bucket has retention of 30d
Setting Retention Policy on gs://k8s-staging-publishing-bot/...
Ensuring the bucket has auto-deletion of 60 days
Setting lifecycle configuration on gs://k8s-staging-publishing-bot/...
Empowering GCS admins
Updated IAM policy for project [k8s-staging-publishing-bot].
bindings:
- members:
  - serviceAccount:service-438481731081@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-publishing-bot@kubernetes.io
  role: roles/viewer
etag: BwWNWq_67bU=
version: 1
No changes made to gs://k8s-staging-publishing-bot/
No changes made to gs://k8s-staging-publishing-bot/
Empowering k8s-infra-staging-publishing-bot@kubernetes.io to GCS
No changes made to gs://k8s-staging-publishing-bot/
No changes made to gs://k8s-staging-publishing-bot/
Done
release-test
Ensuring project exists: k8s-staging-release-test
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-staging-release-test/billingInfo
projectId: k8s-staging-release-test
Empowering k8s-infra-staging-release-test@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-release-test].
bindings:
- members:
  - serviceAccount:service-634027639865@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-release-test@kubernetes.io
  role: roles/viewer
etag: BwWNWrCC_sA=
version: 1
Enabling the container registry API
Ensuring the registry exists and is readable
No changes made to gs://artifacts.k8s-staging-release-test.appspot.com/
Enabling Bucket Policy Only for gs://artifacts.k8s-staging-release-test.appspot.com...
Empowering GCR admins
Updated IAM policy for project [k8s-staging-release-test].
bindings:
- members:
  - serviceAccount:service-634027639865@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-release-test@kubernetes.io
  role: roles/viewer
etag: BwWNWrDYULw=
version: 1
No changes made to gs://artifacts.k8s-staging-release-test.appspot.com/
No changes made to gs://artifacts.k8s-staging-release-test.appspot.com/
Empowering k8s-infra-staging-release-test@kubernetes.io to GCR
No changes made to gs://artifacts.k8s-staging-release-test.appspot.com/
No changes made to gs://artifacts.k8s-staging-release-test.appspot.com/
Enabling the GCS API
Ensuring the bucket exists and is world readable
No changes made to gs://k8s-staging-release-test/
Enabling Bucket Policy Only for gs://k8s-staging-release-test...
Ensuring the bucket has retention of 30d
Setting Retention Policy on gs://k8s-staging-release-test/...
Ensuring the bucket has auto-deletion of 60 days
Setting lifecycle configuration on gs://k8s-staging-release-test/...
Empowering GCS admins
Updated IAM policy for project [k8s-staging-release-test].
bindings:
- members:
  - serviceAccount:service-634027639865@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-release-test@kubernetes.io
  role: roles/viewer
etag: BwWNWrGkG0w=
version: 1
No changes made to gs://k8s-staging-release-test/
No changes made to gs://k8s-staging-release-test/
Empowering k8s-infra-staging-release-test@kubernetes.io to GCS
No changes made to gs://k8s-staging-release-test/
No changes made to gs://k8s-staging-release-test/
Done
```

dims commented:
/hold cancel

thockin commented:
Prod:

```
thockin@freakshow2 k8s.io master /infra/gcp$ ./ensure-prod-storage
ensure-prod-storage-gclb.sh  ensure-prod-storage.sh       
thockin@freakshow2 k8s.io master /infra/gcp$ ./ensure-prod-storage.sh 
Ensuring project exists: k8s-artifacts-prod
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-artifacts-prod/billingInfo
projectId: k8s-artifacts-prod
Enabling the container registry API: k8s-artifacts-prod
Enabling the container analysis API for: k8s-artifacts-prod
Ensuring the registry exists and is readable: k8s-artifacts-prod
region us
No changes made to gs://us.artifacts.k8s-artifacts-prod.appspot.com/
Enabling Bucket Policy Only for gs://us.artifacts.k8s-artifacts-prod.appspot.com...
region eu
No changes made to gs://eu.artifacts.k8s-artifacts-prod.appspot.com/
Enabling Bucket Policy Only for gs://eu.artifacts.k8s-artifacts-prod.appspot.com...
region asia
No changes made to gs://asia.artifacts.k8s-artifacts-prod.appspot.com/
Enabling Bucket Policy Only for gs://asia.artifacts.k8s-artifacts-prod.appspot.com...
Empowering GCR admins: k8s-artifacts-prod
region us
Updated IAM policy for project [k8s-artifacts-prod].
bindings:
- members:
  - serviceAccount:service-388270116193@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-388270116193@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-388270116193@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:388270116193-compute@developer.gserviceaccount.com
  - serviceAccount:388270116193@cloudservices.gserviceaccount.com
  - serviceAccount:service-388270116193@containerregistry.iam.gserviceaccount.com
  - user:justinsb@google.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWrVVQRw=
version: 1
No changes made to gs://us.artifacts.k8s-artifacts-prod.appspot.com/
No changes made to gs://us.artifacts.k8s-artifacts-prod.appspot.com/
region eu
Updated IAM policy for project [k8s-artifacts-prod].
bindings:
- members:
  - serviceAccount:service-388270116193@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-388270116193@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-388270116193@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:388270116193-compute@developer.gserviceaccount.com
  - serviceAccount:388270116193@cloudservices.gserviceaccount.com
  - serviceAccount:service-388270116193@containerregistry.iam.gserviceaccount.com
  - user:justinsb@google.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWrWNtbo=
version: 1
No changes made to gs://eu.artifacts.k8s-artifacts-prod.appspot.com/
No changes made to gs://eu.artifacts.k8s-artifacts-prod.appspot.com/
region asia
Updated IAM policy for project [k8s-artifacts-prod].
bindings:
- members:
  - serviceAccount:service-388270116193@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-388270116193@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-388270116193@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:388270116193-compute@developer.gserviceaccount.com
  - serviceAccount:388270116193@cloudservices.gserviceaccount.com
  - serviceAccount:service-388270116193@containerregistry.iam.gserviceaccount.com
  - user:justinsb@google.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWrXN5mw=
version: 1
No changes made to gs://asia.artifacts.k8s-artifacts-prod.appspot.com/
No changes made to gs://asia.artifacts.k8s-artifacts-prod.appspot.com/
Empowering image promoter to GCR: k8s-artifacts-prod
region us
No changes made to gs://us.artifacts.k8s-artifacts-prod.appspot.com/
No changes made to gs://us.artifacts.k8s-artifacts-prod.appspot.com/
region eu
No changes made to gs://eu.artifacts.k8s-artifacts-prod.appspot.com/
No changes made to gs://eu.artifacts.k8s-artifacts-prod.appspot.com/
region asia
No changes made to gs://asia.artifacts.k8s-artifacts-prod.appspot.com/
No changes made to gs://asia.artifacts.k8s-artifacts-prod.appspot.com/
Ensuring project exists: k8s-artifacts-graveyard
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-artifacts-graveyard/billingInfo
projectId: k8s-artifacts-graveyard
Enabling the container registry API: k8s-artifacts-graveyard
Enabling the container analysis API for: k8s-artifacts-graveyard
Ensuring the registry exists and is readable: k8s-artifacts-graveyard
region us
No changes made to gs://us.artifacts.k8s-artifacts-graveyard.appspot.com/
Enabling Bucket Policy Only for gs://us.artifacts.k8s-artifacts-graveyard.appspot.com...
region eu
No changes made to gs://eu.artifacts.k8s-artifacts-graveyard.appspot.com/
Enabling Bucket Policy Only for gs://eu.artifacts.k8s-artifacts-graveyard.appspot.com...
region asia
No changes made to gs://asia.artifacts.k8s-artifacts-graveyard.appspot.com/
Enabling Bucket Policy Only for gs://asia.artifacts.k8s-artifacts-graveyard.appspot.com...
Empowering GCR admins: k8s-artifacts-graveyard
region us
Updated IAM policy for project [k8s-artifacts-graveyard].
bindings:
- members:
  - serviceAccount:service-969606857874@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-969606857874@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-969606857874@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:969606857874-compute@developer.gserviceaccount.com
  - serviceAccount:969606857874@cloudservices.gserviceaccount.com
  - serviceAccount:service-969606857874@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWrep44U=
version: 1
No changes made to gs://us.artifacts.k8s-artifacts-graveyard.appspot.com/
No changes made to gs://us.artifacts.k8s-artifacts-graveyard.appspot.com/
region eu
Updated IAM policy for project [k8s-artifacts-graveyard].
bindings:
- members:
  - serviceAccount:service-969606857874@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-969606857874@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-969606857874@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:969606857874-compute@developer.gserviceaccount.com
  - serviceAccount:969606857874@cloudservices.gserviceaccount.com
  - serviceAccount:service-969606857874@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWrffGR4=
version: 1
No changes made to gs://eu.artifacts.k8s-artifacts-graveyard.appspot.com/
No changes made to gs://eu.artifacts.k8s-artifacts-graveyard.appspot.com/
region asia
Updated IAM policy for project [k8s-artifacts-graveyard].
bindings:
- members:
  - serviceAccount:service-969606857874@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-969606857874@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-969606857874@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:969606857874-compute@developer.gserviceaccount.com
  - serviceAccount:969606857874@cloudservices.gserviceaccount.com
  - serviceAccount:service-969606857874@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWrglhrQ=
version: 1
No changes made to gs://asia.artifacts.k8s-artifacts-graveyard.appspot.com/
No changes made to gs://asia.artifacts.k8s-artifacts-graveyard.appspot.com/
Empowering image promoter to GCR: k8s-artifacts-graveyard
region us
No changes made to gs://us.artifacts.k8s-artifacts-graveyard.appspot.com/
No changes made to gs://us.artifacts.k8s-artifacts-graveyard.appspot.com/
region eu
No changes made to gs://eu.artifacts.k8s-artifacts-graveyard.appspot.com/
No changes made to gs://eu.artifacts.k8s-artifacts-graveyard.appspot.com/
region asia
No changes made to gs://asia.artifacts.k8s-artifacts-graveyard.appspot.com/
No changes made to gs://asia.artifacts.k8s-artifacts-graveyard.appspot.com/
Ensuring project exists: k8s-cip-test-prod
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-cip-test-prod/billingInfo
projectId: k8s-cip-test-prod
Enabling the container registry API: k8s-cip-test-prod
Enabling the container analysis API for: k8s-cip-test-prod
Ensuring the registry exists and is readable: k8s-cip-test-prod
region us
No changes made to gs://us.artifacts.k8s-cip-test-prod.appspot.com/
Enabling Bucket Policy Only for gs://us.artifacts.k8s-cip-test-prod.appspot.com...
region eu
No changes made to gs://eu.artifacts.k8s-cip-test-prod.appspot.com/
Enabling Bucket Policy Only for gs://eu.artifacts.k8s-cip-test-prod.appspot.com...
region asia
No changes made to gs://asia.artifacts.k8s-cip-test-prod.appspot.com/
Enabling Bucket Policy Only for gs://asia.artifacts.k8s-cip-test-prod.appspot.com...
Empowering GCR admins: k8s-cip-test-prod
region us
Updated IAM policy for project [k8s-cip-test-prod].
bindings:
- members:
  - serviceAccount:service-693665670941@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-693665670941@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-693665670941@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:693665670941-compute@developer.gserviceaccount.com
  - serviceAccount:693665670941@cloudservices.gserviceaccount.com
  - serviceAccount:service-693665670941@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWroPRRc=
version: 1
No changes made to gs://us.artifacts.k8s-cip-test-prod.appspot.com/
No changes made to gs://us.artifacts.k8s-cip-test-prod.appspot.com/
region eu
Updated IAM policy for project [k8s-cip-test-prod].
bindings:
- members:
  - serviceAccount:service-693665670941@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-693665670941@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-693665670941@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:693665670941-compute@developer.gserviceaccount.com
  - serviceAccount:693665670941@cloudservices.gserviceaccount.com
  - serviceAccount:service-693665670941@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWrpMf9o=
version: 1
No changes made to gs://eu.artifacts.k8s-cip-test-prod.appspot.com/
No changes made to gs://eu.artifacts.k8s-cip-test-prod.appspot.com/
region asia
Updated IAM policy for project [k8s-cip-test-prod].
bindings:
- members:
  - serviceAccount:service-693665670941@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-693665670941@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-693665670941@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:693665670941-compute@developer.gserviceaccount.com
  - serviceAccount:693665670941@cloudservices.gserviceaccount.com
  - serviceAccount:service-693665670941@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWrqQNiQ=
version: 1
No changes made to gs://asia.artifacts.k8s-cip-test-prod.appspot.com/
No changes made to gs://asia.artifacts.k8s-cip-test-prod.appspot.com/
Empowering image promoter to GCR: k8s-cip-test-prod
region us
No changes made to gs://us.artifacts.k8s-cip-test-prod.appspot.com/
No changes made to gs://us.artifacts.k8s-cip-test-prod.appspot.com/
region eu
No changes made to gs://eu.artifacts.k8s-cip-test-prod.appspot.com/
No changes made to gs://eu.artifacts.k8s-cip-test-prod.appspot.com/
region asia
No changes made to gs://asia.artifacts.k8s-cip-test-prod.appspot.com/
No changes made to gs://asia.artifacts.k8s-cip-test-prod.appspot.com/
Ensuring project exists: k8s-release-test-prod
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-release-test-prod/billingInfo
projectId: k8s-release-test-prod
Enabling the container registry API: k8s-release-test-prod
Enabling the container analysis API for: k8s-release-test-prod
Ensuring the registry exists and is readable: k8s-release-test-prod
region us
No changes made to gs://us.artifacts.k8s-release-test-prod.appspot.com/
Enabling Bucket Policy Only for gs://us.artifacts.k8s-release-test-prod.appspot.com...
region eu
No changes made to gs://eu.artifacts.k8s-release-test-prod.appspot.com/
Enabling Bucket Policy Only for gs://eu.artifacts.k8s-release-test-prod.appspot.com...
region asia
No changes made to gs://asia.artifacts.k8s-release-test-prod.appspot.com/
Enabling Bucket Policy Only for gs://asia.artifacts.k8s-release-test-prod.appspot.com...
Empowering GCR admins: k8s-release-test-prod
region us
Updated IAM policy for project [k8s-release-test-prod].
bindings:
- members:
  - serviceAccount:service-925892675446@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-925892675446@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:925892675446-compute@developer.gserviceaccount.com
  - serviceAccount:925892675446@cloudservices.gserviceaccount.com
  - serviceAccount:service-925892675446@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWrx6y_I=
version: 1
No changes made to gs://us.artifacts.k8s-release-test-prod.appspot.com/
No changes made to gs://us.artifacts.k8s-release-test-prod.appspot.com/
region eu
Updated IAM policy for project [k8s-release-test-prod].
bindings:
- members:
  - serviceAccount:service-925892675446@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-925892675446@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:925892675446-compute@developer.gserviceaccount.com
  - serviceAccount:925892675446@cloudservices.gserviceaccount.com
  - serviceAccount:service-925892675446@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWryzt0M=
version: 1
No changes made to gs://eu.artifacts.k8s-release-test-prod.appspot.com/
No changes made to gs://eu.artifacts.k8s-release-test-prod.appspot.com/
region asia
Updated IAM policy for project [k8s-release-test-prod].
bindings:
- members:
  - serviceAccount:service-925892675446@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-925892675446@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:925892675446-compute@developer.gserviceaccount.com
  - serviceAccount:925892675446@cloudservices.gserviceaccount.com
  - serviceAccount:service-925892675446@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWrz9JeY=
version: 1
No changes made to gs://asia.artifacts.k8s-release-test-prod.appspot.com/
No changes made to gs://asia.artifacts.k8s-release-test-prod.appspot.com/
Empowering image promoter to GCR: k8s-release-test-prod
region us
No changes made to gs://us.artifacts.k8s-release-test-prod.appspot.com/
No changes made to gs://us.artifacts.k8s-release-test-prod.appspot.com/
region eu
No changes made to gs://eu.artifacts.k8s-release-test-prod.appspot.com/
No changes made to gs://eu.artifacts.k8s-release-test-prod.appspot.com/
region asia
No changes made to gs://asia.artifacts.k8s-release-test-prod.appspot.com/
No changes made to gs://asia.artifacts.k8s-release-test-prod.appspot.com/
Empowering k8s-infra-staging-cip-test@kubernetes.io as project viewer in k8s-cip-test-prod
Updated IAM policy for project [k8s-cip-test-prod].
bindings:
- members:
  - serviceAccount:service-693665670941@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-693665670941@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-693665670941@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:693665670941-compute@developer.gserviceaccount.com
  - serviceAccount:693665670941@cloudservices.gserviceaccount.com
  - serviceAccount:service-693665670941@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-cip-test@kubernetes.io
  role: roles/viewer
etag: BwWNWr3bOFI=
version: 1
Empowering k8s-infra-staging-cip-test@kubernetes.io for GCR in k8s-cip-test-prod
region us
No changes made to gs://us.artifacts.k8s-cip-test-prod.appspot.com/
No changes made to gs://us.artifacts.k8s-cip-test-prod.appspot.com/
region eu
No changes made to gs://eu.artifacts.k8s-cip-test-prod.appspot.com/
No changes made to gs://eu.artifacts.k8s-cip-test-prod.appspot.com/
region asia
No changes made to gs://asia.artifacts.k8s-cip-test-prod.appspot.com/
No changes made to gs://asia.artifacts.k8s-cip-test-prod.appspot.com/
Empowering k8s-infra-staging-release-test@kubernetes.io as project viewer in k8s-release-test-prod
Updated IAM policy for project [k8s-release-test-prod].
bindings:
- members:
  - serviceAccount:service-925892675446@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-925892675446@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:925892675446-compute@developer.gserviceaccount.com
  - serviceAccount:925892675446@cloudservices.gserviceaccount.com
  - serviceAccount:service-925892675446@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-release-test@kubernetes.io
  role: roles/viewer
etag: BwWNWr5vz40=
version: 1
Empowering k8s-infra-staging-release-test@kubernetes.io for GCR in k8s-release-test-prod
region us
No changes made to gs://us.artifacts.k8s-release-test-prod.appspot.com/
No changes made to gs://us.artifacts.k8s-release-test-prod.appspot.com/
region eu
No changes made to gs://eu.artifacts.k8s-release-test-prod.appspot.com/
No changes made to gs://eu.artifacts.k8s-release-test-prod.appspot.com/
region asia
No changes made to gs://asia.artifacts.k8s-release-test-prod.appspot.com/
No changes made to gs://asia.artifacts.k8s-release-test-prod.appspot.com/
Creating GCS bucket gs://k8s-artifacts-prod in k8s-artifacts-prod
Enabling the GCS API
Ensuring the bucket exists and is world readable
No changes made to gs://k8s-artifacts-prod/
Enabling Bucket Policy Only for gs://k8s-artifacts-prod...
Empowering GCS admins
Updated IAM policy for project [k8s-artifacts-prod].
bindings:
- members:
  - serviceAccount:service-388270116193@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:service-388270116193@container-analysis.iam.gserviceaccount.com
  role: roles/containeranalysis.ServiceAgent
- members:
  - serviceAccount:service-388270116193@gcp-sa-containerscanning.iam.gserviceaccount.com
  role: roles/containerscanning.ServiceAgent
- members:
  - serviceAccount:388270116193-compute@developer.gserviceaccount.com
  - serviceAccount:388270116193@cloudservices.gserviceaccount.com
  - serviceAccount:service-388270116193@containerregistry.iam.gserviceaccount.com
  - user:justinsb@google.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWNWr9GmfQ=
version: 1
No changes made to gs://k8s-artifacts-prod/
No changes made to gs://k8s-artifacts-prod/
Configuring the web policy on the bucket
Setting website configuration on gs://k8s-artifacts-prod/...
Copying static content into bucket

WARNING: gsutil rsync uses hashes when modification time is not available at
both the source and destination. Your crcmod installation isn't using the
module's C extension, so checksumming will run very slowly. If this is your
first rsync since updating gsutil, this rsync can take significantly longer than
usual. For help installing the extension, please see "gsutil help crcmod".

Building synchronization state...
Starting synchronization...
Copying file://./static/prod-storage/index.html [Content-Type=text/html]...
/ [1 files][   55.0 B/   55.0 B]                                                
Operation completed over 1 objects/55.0 B.                                       
Done
```
